src/js/Content/Features/Store/Common/FCustomizer.js

```diff
@@ -1,9 +1,18 @@
 import {HTML, Localization, SyncedStorage, TimeUtils} from "../../../../modulesCore";
-import {ContextType, DOMHelper, Feature} from "../../../modulesContent";
+import {ContextType, Feature} from "../../../modulesContent";
 
 export default class FCustomizer extends Feature {
 
     apply() {
+
+        HTML.beforeEnd("#cart_status_data",
```

Comment:
```suggestion
        HTML.afterBegin("#cart_status_data",
```

This ensures that the position of the wishlist button is not changed when our code inserts the customizer button